### ISSUE TITLE
Solutions

### ISSUE THREAD

**Issue by theonlygusti** [Bloxport/Bloxport_Editor#9]
Why don't we build on top of another HTML5 based open source draggable scripting project?

E.g. [Snap](http://snap.berkeley.edu/snapsource/snap.html) or Google's [Blockly](https://github.com/google/blockly).

Or at least use their templates as some sort of reference as we develop our own project.

**Comment by nanalan** [comment]
we shouldn't do that

<sup>because it's a great idea</sup>

**Comment by theonlygusti** [comment]
@nanalan  so you think we should do it?

**Comment by nanalan** [comment]
@theonlygusti of course. Honestly, I don't think that Bloxport is gonna take off, though.

**Comment by theonlygusti** [comment]
@nanalan I left, I gave up on it already xD

**Comment by C0deAMet** [comment]
@theonlygusti So you quit Bloxport?

**Comment by theonlygusti** [comment]
Yeah
